components/content-service-api/typescript/package.json

```diff
@@ -20,8 +20,8 @@
   },
   "devDependencies": {
     "@types/google-protobuf": "^3.2.7",
-    "grpc-tools": "^1.7.2",
-    "grpc_tools_node_protoc_ts": "^2.5.0",
+    "grpc-tools": "^1.10.0",
+    "grpc_tools_node_protoc_ts": "^5.1.0",
```

Comment:
Old version does not generate proper d.ts files, they depended on `grpc` instead of ´@grpc/grpc-js`. The latter we actually use at runtime.